### PR TITLE
More native app experience like for template projects apps (#3951)

### LINKED ISSUE #3951: More native app experience like in app version of template projects

### PR DESCRIPTION
**Pull request by mhrastegari**
Closes #3951

### CODE REVIEW

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Scripts/app.ts [hunk 1]
```diff
@@ -35,3 +35,5 @@ function highlightSnippet() {
 function goToTop() {
     window.scrollTo({ top: 0 });
 }
+
+window.addEventListener('contextmenu', (event) => event.preventDefault());
```
Comment (msynk) [inline src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Scripts/app.ts:39]: we need to discuss about this

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Web/Pages/_Layout.cshtml [hunk 2]
```diff
@@ -48,6 +49,17 @@
     {
         <link rel="manifest" href="manifest.json">
     }
+    @if (isElectron)
+    {
+        <style>
+            * {
```
Comment (msynk) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Web/Pages/_Layout.cshtml:55]: plz move these lines to the below style tags.

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Web/Pages/_Layout.cshtml [hunk 3]
```diff
@@ -47,6 +48,17 @@
     {
         <link rel="manifest" href="manifest.json">
     }
+    @if (isElectron)
+    {
+        <style>
+            * {
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Web/Pages/_Layout.cshtml:54]: plz move these lines to the below style tags.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Web/Pages/_Layout.cshtml (hunk 2 above)
Comment (ysmoradi) [reply src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Web/Pages/_Layout.cshtml:55]: It seems VS bug prevents us from doing that

Comment (msynk) [reply src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Web/Pages/_Layout.cshtml:55]: I see.

File: src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Web/Pages/_Layout.cshtml (hunk 3 above)
Comment (mhrastegari) [reply src/Templates/TodoTemplate/Bit.TodoTemplate/src/Client/Web/Pages/_Layout.cshtml:54]: It's a VS/Framework limitation

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Scripts/app.ts (hunk 1 above)
Comment (mhrastegari) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Scripts/app.ts]: Done

**Review by msynk — APPROVED** [review]

**Pull request merged**